.travis.yml

```diff
@@ -28,13 +26,13 @@ install:
 - pip install git+https://github.com/nerdoftech/s3_index_generator.git
 script:
 - css-validator f5_sphinx_theme/static/css/custom.css
-- htmlhint f5_sphinx_theme
+# - htmlhint f5_sphinx_theme
 - export PATH="$PATH:$(pwd)/node_modules/.bin"
 - webpack
```

Comment:
We should run the webpack command in the test-docs script instead of bagging the container altogether.